stable/external-dns/README.md

```diff
@@ -58,7 +58,13 @@ The following table lists the configurable parameters of the external-dns chart
 | `fullnameOverride`                 | String to fully override external-dns.fullname template with a string                                    | `nil`                                                    |
 | `sources`                          | K8s resources type to be observed for new DNS entries by ExternalDNS                                     | `[service, ingress]`                                     |
 | `provider`                         | DNS provider where the DNS records will be created (mandatory) (options: aws, azure, google, ...)        | `aws`                                                    |
+| `namespace`                        | Limit sources of endpoints to a specific namespace (default: all namespaces)                             | `""`                                                     |
+| `fqdnTemplates`                    | A list of templated string that's used to generate DNS names from sources that don't define a hostname themselves (optional)   | `[]`                               |
+| `combineFQDNAnnotation`            | Combine FQDN template and annotations instead of overwriting (optional)                                  | `false`                                                  |
```

Comment:
Please remove the `(optional)` in all the lines, all of the parameters are supposed to be optional unless they are explicitly marked as mandatory.

Reply:
:+1: https://github.com/helm/charts/pull/17494/commits/8c9dd3bc8861990633e73bd229737583f98eaf8f